src/vivarium_census_prl_synth_pop/results_processing/ssn_and_itin.py

```diff
@@ -1,19 +1,62 @@
-from typing import Any
+from typing import Any, Dict
 
 import numpy as np
 import pandas as pd
+from loguru import logger
+from vivarium import Artifact
 from vivarium.framework.randomness import RandomnessStream
 
+from vivarium_census_prl_synth_pop.results_processing.formatter import (
+    format_data_for_mapping,
+)
 from vivarium_census_prl_synth_pop.utilities import random_integers
 
 
-def generate_itin(
+def get_simulant_id_maps(
```

Comment:
Consider renaming to soemthing like `get_simulant_id_ssn_maps`

Reply:
No, this is meant as a top-level for anything that is indexed by `simulant_id`. Eventually we'll add ITIN.

Reply:
@stevebachmeier The way to think about this is in make_results we are mapping the `simulant_id `column.  If we need to map that column to multiple columns for results (in this case SSNs and ITINs) we need a main runner to get each of the maps from `simulant_id`.